server/src/main/java/org/elasticsearch/search/retriever/CompoundRetrieverBuilder.java

```diff
@@ -46,6 +47,8 @@
  */
 public abstract class CompoundRetrieverBuilder<T extends CompoundRetrieverBuilder<T>> extends RetrieverBuilder {
 
+    public static final NodeFeature INNER_RETRIEVERS_FILTER_SUPPORT = new NodeFeature("inner_retrievers_filter_support");
```

Comment:
If this is just for testing, it should be a capability.

Reply:
Yeap I added it just for testing. It is used only through the `getTestFeatures` override; isn't this the intended approach for the test method (it's still not 100% clear to me tbh). Should i still update this to make use of capabilities instead?

Reply:
eh, since its only a test feature, I guess it doesn't really matter.